state/pg_state_manager.go

```diff
@@ -1929,3 +1929,51 @@ func (arr Capabilities) Value() (driver.Value, error) {
 	}
 	return fmt.Sprintf("{%s}", strings.Join(arr, ",")), nil
 }
+
+func (sm *SQLStateManager) GetRunStatus(runID string) (RunStatus, error) {
+	var status RunStatus
+	tx, err := sm.db.Begin()
+	if err != nil {
+		return status, errors.Wrap(err, "failed to begin transaction")
+	}
+	_, err = tx.Exec("SET LOCAL lock_timeout = '500ms'")
+	if err != nil {
+		tx.Rollback()
+		return status, errors.Wrap(err, "failed to set lock timeout")
+	}
+	err = tx.QueryRow(GetRunStatusSQL, runID).Scan(
+		&status.RunID,
+		&status.DefinitionID,
+		&status.Alias,
+		&status.ClusterName,
+		&status.Status,
+		&status.QueuedAt,
+		&status.StartedAt,
+		&status.FinishedAt,
+		&status.ExitCode,
+		&status.ExitReason,
```

Comment:
These two Exit fields are pointers to string, does that mean the syntax here should elide the ampersand character?